ballerina/modules/parser/tests/parser_tests.bal

```diff
@@ -533,20 +533,20 @@ isolated function testVariables() returns error? {
 }
 isolated function testNonNullTypeVariables() returns error? {
     string document = "query getId($name: String!, $age: Int!) { profile(userName:$name, userAge:$age) { id } }";
```

Comment:
```suggestion
    string document = "query getId($name:String!, $age:Int!) { profile(userName:$name, userAge:$age) { id } }";
```

Reply:
Is this really needed? There is a space before the Type in GraphQL documents, right?